### PR TITLE
Leo/tp/help button

### PR DESCRIPTION
**Pull request by leonardo-fan**
- added new help button that shows popover with gifs on how to use the term planner
- hover to play 
- added extra gif on option header from initial wireframe but not sure if needed
- grid layout as one column doesn't utilise screen space as much
help-button-demo

### CODE REVIEW

**Comment by Peedee2002** [comment]
do you have a video of your feature in action?

**Comment by leonardo-fan** [comment]
> do you have a video of your feature in action?

sure just added a gif of it if that's fine

File: frontend/src/pages/TermPlanner/optionsHeader/HelpMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          3. Hide Year
+          </Title>
+        </div>
+        <img 
+          className="helpGif" 
+          onMouseEnter={() => {setHideYearPlaying(true)}} 
+          onMouseLeave={() => {setHideYearPlaying(false)}}
+          src={hideYearPlaying ? hideYearGif : hideYearPng}
+          alt="hide year walkthrough"
+          style={hideYearPlaying ? {boxShadow: "0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)"} : {}}
```
Comment (Peedee2002) [inline frontend/src/pages/TermPlanner/optionsHeader/HelpMenu.jsx]: probably make boxShadow a constant defined on the top if it cant be a CSS style

**Review by Peedee2002 — APPROVED** [review]
very nice, just a small change

File: frontend/src/pages/TermPlanner/optionsHeader/HelpMenu.jsx (hunk 1 above)
Comment (leonardo-fan) [reply frontend/src/pages/TermPlanner/optionsHeader/HelpMenu.jsx:71]: ohh yeah, I might make it a css style instead with a diff classname

Comment (leonardo-fan) [reply frontend/src/pages/TermPlanner/optionsHeader/HelpMenu.jsx]: ye good point - think I'll make it a css style instead

**Pull request merged**